lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -5551,17 +5553,17 @@ void AdjointEmitter::accumulateIndirect(
   }
   case VectorSpace::Kind::Function: {
     llvm_unreachable(
-        "Unimplemented: Emit thunks for abstracting adjoint accumulation");
+        "Unimplemented: Emit thunks for abstracting pullback accumulation");
```

Comment:
Revert; "adjoint accumulation" here refers to "adjoint value accumulation".
"Pullback accumulation" kind of makes sense, as in "accumulation in the pullback function". But it makes less sense to me.
```suggestion
        "Unimplemented: Emit thunks for abstracting adjoint value accumulation");
```